lib/model/content.dart

```diff
@@ -1028,21 +1053,46 @@ class _ZulipContentParser {
       final grandchild = child.nodes[0];
       if (grandchild is! dom.Element) return null;
       if (grandchild.localName != 'img') return null;
-      if (grandchild.className.isNotEmpty) return null;
-      return grandchild;
+      return (child, grandchild);
     }();
 
     final debugHtmlNode = kDebugMode ? divElement : null;
-    if (imgElement == null) {
+    if (elements == null) {
       return UnimplementedBlockContentNode(htmlNode: divElement);
     }
 
+    final (linkElement, imgElement) = elements;
+    if (imgElement.className == 'image-loading-placeholder') {
+      return ImageNode(
+        srcUrl: null,
+        thumbnailUrl: null,
+        loading: true,
+        debugHtmlNode: debugHtmlNode);
```

Comment:
~~so in particular:~~
```suggestion
      return ImageNode.loading(debugHtmlNode: debugHtmlNode);
```